src/main/java/Javatro/Round.java

```
@@ -76,14 +85,14 @@ public void playCards(int[] cardIndices) {
      * @param cardIndices Indices of cards to discard from the holding hand
      * @throws IllegalStateException If no remaining discards are available
      */
-    public void discardCards(int[] cardIndices) {
+    public void discardCards(List<Integer> cardIndices) {
         if (remainingDiscards <= 0) {
             throw new IllegalStateException("No remaining discards available");
```

this exception is not handled